src/Calculator/GraphingSettings.xaml.h

```diff
@@ -0,0 +1,55 @@
+//
+// MyUserControl.xaml.h
+// Declaration of the MyUserControl class
+//
+
+#pragma once
+
+#include "..\..\src\GraphingInterfaces\IMathSolver.h"
```

Comment:
This feels off to me. doesn't seem like we should be going out to the src folder to grab header files from other projects, especially since other parts of the code that currently reference these header files will actually be pointing to a different header file because of the override props file.

Reply:
Unneeded, and removed.